Helper/ModuleRetriever.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
```

Comment:
@Jiyuan-Bolt  The copyright should be 2020, right?

Reply:
@johnmccombs1 This is the date of copyright being prepared. Daniel Dresser confirmed with our legal team that it is the correct one.